config/locales/api/help/upcoming/om_ET.html

```diff
@@ -244,70 +270,70 @@ <h3>&ldquo;bu&rsquo;aa bilbilaa&rdquo; milikkiteessi.</h3>
               <p>Jalaqaba meeshaakee lamkaasuuf yaali. Yoo Simple cabaa jira ta&rsquo;eefi sirreessuu hin dandeenye ta&rsquo;e, gara fuula <span class="help">Gargaarsaa</span> deemiitii to&rsquo;ataakee ykn tuuta keenya quunnami.</p>
             </div>
         </div>
-        
+
         <div class="card">
 <a class="faq">Kaastuun hojjetaa hin jiru. Maal gochuun qaba?</a><div class="answer">
               <p>Fuula argii gara jalaa irratti lakkoofsa Paaspoortii Dhiibbaa dhiigaa galchi. Yoo rakkoon kun itti fufe, bilbilakee lamkaasi. Amma shin hojjetu yoo ta&rsquo;e, to&rsquo;ataakee quunnami.</p>
             </div>
         </div>
-        
+
         <div class="card">
 <a class="faq">Akkamittan fayyadamakoo foyyessuu danda&rsquo;a?</a><div class="answer">
               <p>Fayyadamakee sajoo qindeessituu cuqaasuudhaan <span class="code">Foyyeessuu</span> isa jedhu tuqii ammayoomsi. Simple dhiyeenya ammayyoomee meeshaakee irratti qabaachuun baay&rsquo;ee barbaachisaadha.</p>
               <div class="help-example">%{icon_settings_svg}</div>
             </div>
         </div>
-        
+
         <div class="card">
 <a class="faq">Dhukkubsataa hordoffii irra jiru argachuu hin dandeenye</a><div class="answer">
               <p>Si&rsquo;a tokko tokko, dhukkubsattoota hordoffii irra jiran,yammuu jarri Paaspoortii Dhiibbaa dhiigaa hin qabne, barbaaduun ni cima. Daqiiqaa 5 booda, dhukkubsataa argachuu yoo dadhabde, akka dhukkubsataa haaraatti galmeessi. Gara fuulduraatti galmee dhukkubsataa qindeessuun ni danda&rsquo;ama.</p>
             </div>
         </div>
-        
+
         <div class="card">
 <a class="faq">Dhukkubsataan tokko Paaspoortii Dhiibbaa dhiigaa lama qabaachuu ni danda&rsquo;a?</a><div class="answer">
               <p>Eeyyee. Dhukkubsataan tokko Paaspoortii Dhiibbaa dhiigaa baay&rsquo;ee qabaachuu ni danda&rsquo;a. Hundumti isaanii hojjechuu itti fufu.</p>
             </div>
         </div>
-        
+
         <div class="card">
 <a class="faq">Paaspoortii Dhiibbaa dhiigaan fixe.</a><div class="answer">
               <p>Utuu hin fixin dura, Paaspoortii Dhiibbaa dhiigaa dabalataa akka ajajuuf to&rsquo;ataakee gaafadhu. Yaadadhu, osoodhuma Paaspoortii Dhiibbaa dhiigaa hin qabaatin, dhukkubsattoota galmeessitee boodarra Paaspoortii kennuufi dandeessa.</p>
             </div>
         </div>
-        
+
         <div class="card">
 <a class="faq">&ldquo;Sync&rdquo;/&rsquo;wal-simni&rsquo; maaliidhaa?</a><div class="answer">
               <p>&ldquo;Wal-simni&rdquo;/&rdquo; sync&rdquo; deetaa kee kaadima giddu-galeessaa irra siif kaa&rsquo;a. Galmeen kee yoo xiqqaate guyyaatti al-tokko ol-ka&rsquo;aamuu isaa mirkaneessi. Yoo quunnamtii interneetii gaarii hin qabdu ta&rsquo;e, wal-simiinsi deetaa kee yeroo muraasa fudhachuu danda&rsquo;a.</p>
-              <div class="help-example">%{gargaarsa_wal-sima_svg}</div>
+              <div class="help-example">%{help_sync_svg}</div>
```

Comment:
What is this? Did we end up getting a translated file name?

Reply:
Yup, the Oromo translations translated the asset names inside our interpolation too. I just rectified them manually instead of asking GPI to do another round of translation on them.